src/solver/simulation/CMakeLists.txt

```diff
@@ -45,15 +45,16 @@ set(SRC_SIMULATION
         adequacy_patch_runtime_data.cpp
         include/antares/solver/simulation/ITimeSeriesNumbersWriter.h
         TimeSeriesNumbersWriter.cpp
-		include/antares/solver/simulation/BindingConstraintsTimeSeriesNumbersWriter.h
-		include/antares/solver/simulation/ISimulationObserver.h
+        include/antares/solver/simulation/BindingConstraintsTimeSeriesNumbersWriter.h
+        include/antares/solver/simulation/ISimulationObserver.h
 )
 source_group("simulation" FILES ${SRC_SIMULATION})
 
 #
 # --- Library VARIABLES ---
 #
 add_library(antares-solver-simulation)
+add_library(Antares::antares-solver-simulation ALIAS antares-solver-simulation)
```

Comment:
```suggestion
add_library(Antares::solver-simulation ALIAS antares-solver-simulation)
```

Reply:
Also, where is this alias needed, if at all ?

Reply:
https://gopro-collaboratif.rte-france.com/pages/viewpage.action?pageId=411479146
Section: "Gestion des targets dans Simulator "

Reply:
I understand the suggestion but the names need to be the same